.bazelignore

```diff
@@ -23,3 +23,4 @@ node_modules/
 npm/private/test/node_modules/
 npm/private/test/npm_package/node_modules/
 npm/private/test/npm_package_publish/node_modules
+.git
```

Comment:
I don't think I've ever seen this in any bazel repo, do you have any more info on this?